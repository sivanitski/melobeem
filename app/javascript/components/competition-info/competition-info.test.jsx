import React from "react";
import renderer from "react-test-renderer";

import CompetitionInfo from "./competition-info";

const mockProps = {
  timeLeft: `1995-12-17T03:24:00`,
  prize: `105`,
};

describe("competition-info component", () => {
  it("should render competition-info component", () => {
    const component = renderer
      .create(
        <CompetitionInfo
          timeLeft={mockProps.timeLeft}
          prize={mockProps.prize}
        />
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
