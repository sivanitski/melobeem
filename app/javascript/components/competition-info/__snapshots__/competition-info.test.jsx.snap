// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`competition-info component should render competition-info component 1`] = `
<div
  className="competition-info"
>
  <div
    className="competition-info__item"
  >
    <div
      className="competition-info__item__title text-grey"
    >
      Days left
    </div>
    <div
      className="competition-info__item__title headline--medium text-pink"
    >
      -9179
    </div>
  </div>
  <div
    className="competition-info__item"
  >
    <div
      className="competition-info__item__title text-grey"
    >
      Prize
    </div>
    <div
      className="competition-info__item__title headline--medium text-pink"
    >
      Â£
      105
    </div>
  </div>
</div>
`;
